static_src/test/unit/stores/service_plan_store.spec.js

```diff
@@ -137,6 +137,7 @@ describe('ServicePlanStore', function() {
 
     it('should set fetching to false, fetched to true', function() {
       ServicePlanStore.fetching = true;
+      ServicePlanStore.waitingOnRequests = false;
```

Comment:
Cool, that makes sense. Maybe its something we can refactor into the base store since there are a few stores that have to be aware of multiple requests.